### ISSUE TITLE
Blender 4.0

### ISSUE THREAD

**Issue by CS1351102-Jones-Matthew** [stuffmatic/fSpy#138]
I was trying to install this and tried following the instructions and still couldn't get it to work. I am on Blender 4.0 and was wondering if it has been ported to this version yet, or if that needs to still happen.

**Comment by tzzawadzki** [comment]
Same here, doesn't work at all.

**Comment by iulian3144** [comment]
Just tried it yesterday on Windows and it works just fine for me with the following versions:
```
Blender 4.0.2
fSpy 1.0.3
fSpy-Blender 1.0.3
```

What, exactly, is not working? Is the `.fspy` import not working at all, camera is not configured as expected etc..

![image](https://github.com/stuffmatic/fSpy/assets/12052955/9eecab06-5318-4f7e-b5f2-714255970f48)
I've just imported the `.fspy` file in Blender (`File` > `Import` > `fSpy (.fspy)`) using fspy-Blender addon.

**Comment by NudiDesign** [comment]
fspy 1.0.3
Blender 4: bpy.ops.object.camera_add.poll() failed, context is incorrect
Blender 3.6: same .fspy file works fine.

**Comment by iulian3144** [comment]
[This](https://github.com/stuffmatic/fSpy-Blender/issues/18) seems to be related.
TLDR: Make sure you're in object mode.

If this is still not working, I would try the import using a different blender file.
